tests/aws/test_athena.py

```diff
@@ -5,13 +5,15 @@
 from pandas import DataFrame
 
 import hip_data_tools.aws.athena as athena
-from hip_data_tools.aws.athena import AthenaUtil
+from hip_data_tools.aws.athena import AthenaUtil, AthenaSettings
 
 
 class TestAthenaUtil(TestCase):
     @classmethod
     def setUpClass(cls):
-        cls.au = AthenaUtil(database="test", conn=None)
+        cls.au = AthenaUtil(
+            settings=AthenaSettings(database="test", conn=None, output_bucket=None,
+                                    output_key=None))
```

Comment:
should write a test to make sure it is not None

Reply:
Added a unit test